keyset.go

```diff
@@ -249,19 +259,28 @@ func initkeyset(keysetName string, Org []string) error {
 	caTemplate.BasicConstraintsValid = true
 
 	// Generate the manifest rootCA
-	err = generaterootCA(filepath.Join(keysetPath, "manifest-ca"), &caTemplate)
+	err = generaterootCA(filepath.Join(keysetPath, "manifest-ca"), &caTemplate, false)
```

Comment:
I hate to create more work here, but when someone comes back to look at this in a week, they'll ask themselves "what does false mean?".

You can create a type aliased to bool for this, but actually you don't even need to do that.  You can just define

const (
        doGuid = true
        noGuid = false
)

and then pass doGuid/noGuid to generaterootCA

Reply:
thanks! done.